tests/test_integration/test_wf_create_case_metadata.py

```diff
@@ -24,6 +35,66 @@ def test_create_case_metadata_runs_successfully(fmu_snakeoil_project, monkeypatc
 
     with open(fmu_case_yml, encoding="utf-8") as f:
         fmu_case = yaml.safe_load(f)
+
     assert fmu_case["fmu"]["case"]["name"] == "snakeoil"
     assert fmu_case["fmu"]["case"]["user"]["id"] == "user"
     assert fmu_case["source"] == "fmu"
+    assert len(fmu_case["tracklog"]) == 1
+    assert fmu_case["tracklog"][0]["user"]["id"] == getpass.getuser()
+
+
+def test_create_case_metadata_warns_without_overwriting(
+    fmu_snakeoil_project, monkeypatch, mocker
+):
+    monkeypatch.chdir(fmu_snakeoil_project / "ert/model")
+    _add_create_case_workflow("snakeoil.ert")
```

Comment:
Might be reasonable to add these into fmu_snakeoil_case... but other integration tests might not need or want these. If they mostly seem to, will add it there